skills/common/process/merge/runtime/git-operations.sh

```
@@ -99,6 +99,55 @@ merge_rebase() {
+    _branch="$2"
+    _expected_remote="$3"
+    _expected_local="$4"
+    _remote="$5"
+    [ -n "$_remote" ] || _remote="origin"
+
+    if [ ! -d "$_worktree" ] || ! git -C "$_worktree" rev-parse --git-dir >/dev/null 2>&1; then
+        echo "ERROR: worktree path is not a git worktree" >&2
+        return 1
+    fi
+    case "$_branch" in
+        ""|main|refs/heads/main|-*|*..*|*/../*|*/.|*/.) echo "ERROR: protected or invalid target branch" >&2; return 1 ;;
```

Shellcheck (0.11.0)

[warning] This pattern always overrides a later one on line 117.

(SC2221)